_topic_maps/_topic_map_osd.yml

```diff
@@ -107,6 +104,87 @@ Topics:
 - Name: Deleting an OpenShift Dedicated cluster
   File: osd-deleting-a-cluster
 ---
+Name: Support
+Dir: support
+Distros: openshift-dedicated
+Topics:
+- Name: Support overview
+  File: index
+- Name: Managing your cluster resources
+  File: managing-cluster-resources
+- Name: Getting support
+  File: getting-support
+  Distros: openshift-dedicated
+- Name: Remote health monitoring with connected clusters
+  Dir: remote_health_monitoring
+  Distros: openshift-dedicated
+  Topics:
+  - Name: About remote health monitoring
+    File: about-remote-health-monitoring
+  - Name: Showing data collected by remote health monitoring
+    File: showing-data-collected-by-remote-health-monitoring
+# cannot get resource "secrets" in API group "" in the namespace "openshift-config"
+#  - Name: Opting out of remote health reporting
+#    File: opting-out-of-remote-health-reporting
+# cannot get resource "secrets" in API group "" in the namespace "openshift-config"
+#  - Name: Enabling remote health reporting
+#    File: enabling-remote-health-reporting
+  - Name: Using Insights to identify issues with your cluster
+    File: using-insights-to-identify-issues-with-your-cluster
+  - Name: Using Insights Operator
+    File: using-insights-operator
+# Not supported per Michael McNeill
+# - Name: Using remote health reporting in a restricted network
+#   File: remote-health-reporting-from-restricted-network
+# cannot list resource "secrets" in API group "" in the namespace "openshift-config"
+#  - Name: Importing simple content access entitlements with Insights Operator
+#    File: insights-operator-simple-access
+# must-gather not supported for customers, per Dustin Row, cannot create resource "namespaces" 
+# - Name: Gathering data about your cluster
```

Comment:
I see changes in this assembly and module below, but they aren't included. Should this be commented in?

Reply:
No everything in this assembly is done by RH for the customer.